config/faker-generator.js

```diff
@@ -0,0 +1,18 @@
+const _ = require('underscore');
+const fs = require('fs');
+const files = fs.readdirSync('./test/fixtures/config');
+
+_.each(files, (file) => {
+  const entity_factory = require(`../test/fixtures/config/${file}`);
+  const dest = `./test/fixtures/collections/${file.split('.')[0]}.json`;
+
+  const collection = _.times(1000, () => {
+    return entity_factory.generate();
+  });
+
+  fs.writeFile(dest, JSON.stringify(collection), (e) => {
+    if(e) {
```

Comment:
The config files aren't going to fail the linter during build, but I think `npm run lint` should still error for the config files.  Needs to be run as the style is different